ioc_copier/ioc_copier.py

```diff
@@ -156,9 +191,21 @@ def get_file_text(file, ioc, root):
     :return:
         The text from the file, with the IOC number updated to the current copy.
     """
-    with open(os.path.join(root, file), "r") as file_pointer:
+    path = os.path.join(root, file)
+    with open(path, "r") as file_pointer:
         text = file_pointer.readlines()
+    final_lines = []
+    if START_COPY == 1:
+        if file == "st.cmd":
+            final_lines = [text.pop(-1)]
```

Comment:
This seems error-prone - can't we detect the line that calls `st-common.cmd` in a better way than this?

Reply:
replaced with check for call to st-common and add to list of indexes to skip